doc/releases/octopus.rst

```diff
@@ -124,9 +124,18 @@ RADOS
 **RGW** object storage
 ~~~~~~~~~~~~~~~~~~~~~~
   
-* Multi-site replication can now be managed on a per-bucket basis (EXPERIMENTAL).
-* WORM?
-* bucket tagging?
+* New `Multisite Sync Policy`_ primitives for per-bucket replication. (EXPERIMENTAL)
+* S3 feature support:
+    - Bucket Replication (EXPERIMENTAL)
+    - Bucket Tagging
+    - Object Lock
+    - Public Access Block
```

Comment:
maybe worth mentioning `for buckets`

Reply:
List Objects V2

Reply:
nvm list v2 was already backported in nautilus

Reply:
> maybe worth mentioning `for buckets`

added thanks